README.md

```diff
@@ -70,7 +70,12 @@ the server is restarted, reconnect to the system to perform AGW configuration.
 
 ### 2. Configure
 
-Fetch `rootCA.pem` certificate from Orchestrator, upload it to the Access Gateway host and execute:
+Fetch `rootCA.pem` certificate from Orchestrator deployment:
+```bash
+juju scp --container="magma-orc8r-certifier" orc8r-certifier/0:/var/opt/magma/certs/rootCA.pem rootCA.pem
```

Comment:
IMO this should go to the `magma-access-gateway-operator` rather than here. Snap is completely independent from Juju and can be used by people who have absolutely no idea about it. 
WDYT?

Reply:
Yeah, actually I agree. I add it to the charm and can either remove it or leave it as a note, something like:
"If you are using charmed-orc8r, you can retrieve the certificate like so..." Let me know what you think

Reply:
I'd say drop it completely, but perhaps others have different view.

Reply:
Yeah I agree with @Gmerold here

Reply:
I'll close the PR.